components/progressTracker/ProgressStep.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import Box from '../box';
+import { TextSmall } from '../typography';
+
+class ProgressStep extends PureComponent {
```

Comment:
I guess in order to be conventionally correct this should be in another folder? `Modules/ProgressStep.js`?

Reply:
@InstaK I was following the same structure as we have for the Datagrid component where child components are placed in the same directory as the main component.

Reply:
Fair enough, not sure if we should change this (because everywhere else I've seen it it was like I just mentioned or just keep going this way 🤔 )

Reply:
Not sure what you're saying. `Modules` isn't even a directory at the moment?

Reply:
@lowiebenoot if we are talking about specific child components for parent components, they are not placed in a separate folder.